### PR TITLE
add ebpf 2.0 blog post

### PR DESCRIPTION
**Pull request by xmulligan**
Don't merge until the launch date

### CODE REVIEW

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+path: '/blog/launching-excel-bpf'
+date: '2023-04-017T10:00:00.000Z'
```
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:3]:
```suggestion
date: '2023-04-01T10:00:00.000Z'
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Looking at where eBPF 1.0 has transformed the world shows us some of the potential excelBPF has for the future. In the cloud native world, [eBPF has unlocked innovation](https://www.infoworld.com/article/3689690/how-ebpf-unlocks-cloud-native-innovation.html) by shortening production feedback loops, extended functionality with a safety belt, provided new building blocks while still leveraging existing tooling, allowing for on the fly upgrades, making data processing faster with lower resource consumption, and creating better abstractions for observability and security. excelBPF is bringing these benefits outside the IT department. Spreadsheets no longer need to be held back by decades old macros.
```
Comment (qmonnet) [inline content/blog-posts/2023-04-01-excelbpf/index.md:20]: The first one is a long sentence, with a mix of (at the beginning) present perfect and (near the end) -ing forms. Maybe split in two?

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+“With eBPF 1.0, [Cilium](cilium.io) was able to transform the cloud native infrastructure landscape in areas as diverse as modernizing telecommunications networks, bringing retail stores online, and moving money through markets. Our end users are excited at what excelBPF will bring to the rest of the organization. Just as prompt engineering has taken off with the launch of large language models, I think we soon see job advertisements for excelBPF engineers who are able to change spreadsheets from individual cells to hives of activity.” said Thomas Graf co-founder of the Cilium project.
```
Comment (qmonnet) [inline content/blog-posts/2023-04-01-excelbpf/index.md:22]:
```suggestion
“With eBPF 1.0, [Cilium](cilium.io) was able to transform the cloud native infrastructure landscape in areas as diverse as modernizing telecommunications networks, bringing retail stores online, and moving money through markets. Our end users are excited at what excelBPF will bring to the rest of the organization. Just as prompt engineering has taken off with the launch of large language models, I think we soon see job advertisements for excelBPF engineers who are able to change spreadsheets from individual cells to hives of activity.” said Thomas Graf, co-founder of the Cilium project.
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+With money flowing this quickly through excelBPF, you could easily guess that the funding for companies is just behind it and you wouldn’t be wrong. “After seeing how fast Isovalent changed the cloud native landscape with eBPF 1.0, we have an open checkbook for companies looking to build around excelBPF. I'm literally just waiting for those excelBPF financial models to hit my inbox.” said Martanique Casandro, Partner at Redwood Tree Venturez (r17z).
+
+The [eBPF Landscape](https://ebpf.io/applications/) is rapidly evolving and we are excited to see the first excelBPF powered applications be added to the landscape. Be sure to check back regularly to see how the space is evolving. If you want to catch up on eBPF 1.0 before diving into excelBPF, check out [what is eBPF](https://ebpf.io/what-is-ebpf/), the [getting started page](https://ebpf.io/get-started/), and our [interactive eBPF labs](https://play.instruqt.com/embed/isovalent/tracks/ebpf-getting-started?token=em_9nxLzhlV41gb3rKM&show_challenges=true).
```
Comment (qmonnet) [inline content/blog-posts/2023-04-01-excelbpf/index.md:28]: Link to the eCHO newsletter as well? Or it's easy to find from the get-started page probably

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+“With eBPF 1.0, [Cilium](cilium.io) was able to transform the cloud native infrastructure landscape in areas as diverse as modernizing telecommunications networks, bringing retail stores online, and moving money through markets. Our end users are excited at what excelBPF will bring to the rest of the organization. Just as prompt engineering has taken off with the launch of large language models, I think we soon see job advertisements for excelBPF engineers who are able to change spreadsheets from individual cells to hives of activity.” said Thomas Graf co-founder of the Cilium project.
+
+Early versions of excelBPF were put in the hands of the community, and feedback has been impressive. “Forget about ChatGTP, FTX and SVB, excelBPF has completely stopped the press for me. As someone striving to be the living embodiment of "Operational Excellence", excelBPF is the only tool I need in my arsenal. Everything one needs spanning FinOps, HROps, SalesOps, RevOps, BizOps, CloudOps and even OpsOps is now just an excelBPF add-on model away. I finally have some time to pretend like I'm going surfing again!” said Christiaan Kuun, Head of Operations at Isovalent.
```
Comment (lizrice) [inline content/blog-posts/2023-04-01-excelbpf/index.md:24]:
```suggestion
Early versions of excelBPF were put in the hands of the community, and feedback has been impressive. “Forget about ChatGPT, FTX and SVB, excelBPF has completely stopped the press for me. As someone striving to be the living embodiment of "Operational Excellence", excelBPF is the only tool I need in my arsenal. Everything one needs spanning FinOps, HROps, SalesOps, RevOps, BizOps, CloudOps and even OpsOps is now just an excelBPF add-on model away. I finally have some time to pretend like I'm going surfing again!” said Christiaan Kuun, Head of Operations at Isovalent.
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+toc: false
+categories:
+  - Community
+---
+
+Author: Bill Mulligan - Isovalent
+
+Today the eBPF community is excited to announce the launch of the next version of [eBPF](https://ebpf.io/), excelBPF. Excel is the most widely used programming language in the world and underpins the functioning of many of the most critical institutions in the world. Bringing power of BPF to spreadsheets will enable millions of people to leverage this production proven technology, making bytecode only one cell away.
+
+eBPF has been deployed at scale for over half a decade now, delivering you every cat picture on Facebook, processing and securing your Google queries, and enabling you to binge shows on Netflix without waiting for it to load. These initial adopters showed us the impact eBPF can have on performance and innovation in areas like networking, observability, security, tracing, and profiling. excelBPF will now bring these benefits to many new use cases like finance, inventory management, consulting, and planning your next vacation.
```
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:16]:
```suggestion
eBPF has been deployed at scale for over half a decade now, delivering you every cat picture on Facebook, processing and securing your Google queries, and enabling you to binge shows on Netflix without waiting for it to load. These initial adopters showed us the impact eBPF can have on performance and innovation in areas like networking, observability, security, tracing, and profiling. **excelBPF™** will now bring these benefits to many new use cases like finance, inventory management, consulting, and planning your next vacation.
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+“When we got the first version of eBPF merged into the Linux kernel in 2014, we had ideas about how transformative the technology could be for the Linux kernel, but they were still to be proven out. Seeing the massive step change in innovation and flexibility eBPF has enabled for the kernel community, I’m excited to see what excelBPF holds in store for the world. I think we are really just scratching the surface of what is possible. It feels like Day 1 all over again.” said Daniel Borkmann, co-creator of eBPF.
```
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:18]:
```suggestion
“When we got the first version of eBPF merged into the Linux kernel in 2014, we had ideas about how transformative the technology could be for the Linux kernel, but they were still to be proven out. Seeing the massive step change in innovation and flexibility eBPF has enabled for the kernel community, I’m excited to see what **excelBPF™** holds in store for the world. I think we are really just scratching the surface of what is possible. It feels like Day 1 all over again.” said Daniel Borkmann, co-creator of eBPF.
```

File: content/blog-posts/2023-04-01-excelbpf/index.md (hunk 3 above)
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:22]:
```suggestion
“With eBPF 1.0, [Cilium](cilium.io) was able to transform the cloud native infrastructure landscape in areas as diverse as modernizing telecommunications networks, bringing retail stores online, and moving money through markets. Our end users are excited at what **excelBPF™** will bring to the rest of the organization. Just as prompt engineering has taken off with the launch of large language models, I think we soon see job advertisements for excelBPF engineers who are able to change spreadsheets from individual cells to hives of activity.” said Thomas Graf, co-founder of the Cilium project.
```

File: content/blog-posts/2023-04-01-excelbpf/index.md (hunk 5 above)
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:24]:
```suggestion
Early versions of **excelBPF™** were put in the hands of the community, and feedback has been impressive. “Forget about ChatGPT, FTX and SVB, **excelBPF™** has completely stopped the press for me. As someone striving to be the living embodiment of "Operational Excellence", **excelBPF™** is the only tool I need in my arsenal. Everything one needs spanning FinOps, HROps, SalesOps, RevOps, BizOps, CloudOps and even OpsOps is now just an **excelBPF™** add-on model away. I finally have some time to pretend like I'm going surfing again!” said Christiaan Kuun, Head of Operations at Isovalent.
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+Early versions of excelBPF were put in the hands of the community, and feedback has been impressive. “Forget about ChatGTP, FTX and SVB, excelBPF has completely stopped the press for me. As someone striving to be the living embodiment of "Operational Excellence", excelBPF is the only tool I need in my arsenal. Everything one needs spanning FinOps, HROps, SalesOps, RevOps, BizOps, CloudOps and even OpsOps is now just an excelBPF add-on model away. I finally have some time to pretend like I'm going surfing again!” said Christiaan Kuun, Head of Operations at Isovalent.
+
+With money flowing this quickly through excelBPF, you could easily guess that the funding for companies is just behind it and you wouldn’t be wrong. “After seeing how fast Isovalent changed the cloud native landscape with eBPF 1.0, we have an open checkbook for companies looking to build around excelBPF. I'm literally just waiting for those excelBPF financial models to hit my inbox.” said Martanique Casandro, Partner at Redwood Tree Venturez (r17z).
```
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:26]:
```suggestion
With money flowing this quickly through **excelBPF™**, you could easily guess that the funding for companies is just behind it and you wouldn’t be wrong. “After seeing how fast Isovalent changed the cloud native landscape with eBPF 1.0, we have an open checkbook for companies looking to build around **excelBPF™**. I'm literally just waiting for those **excelBPF™** financial models to hit my inbox.” said Martanique Casandro, Partner at Redwood Tree Venturez (r17z).
```

File: content/blog-posts/2023-04-01-excelbpf/index.md (hunk 4 above)
Comment (xmulligan) [reply content/blog-posts/2023-04-01-excelbpf/index.md:28]:
```suggestion
The [eBPF Landscape](https://ebpf.io/applications/) is rapidly evolving and we are excited to see the first **excelBPF™** powered applications be added to the landscape. Be sure to check back regularly to see how the space is evolving. If you want to catch up on eBPF 1.0 before diving into excelBPF, check out [what is eBPF](https://ebpf.io/what-is-ebpf/), the [getting started page](https://ebpf.io/get-started/), our [interactive eBPF labs](https://play.instruqt.com/embed/isovalent/tracks/ebpf-getting-started?token=em_9nxLzhlV41gb3rKM&show_challenges=true), and subscribe to [eCHO News](https://ebpf.io/newsletter/) to keep up on all the updates from the eBPF community.
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+description: 'Hear about the latest innovation from the eBPF community, turn your cells into hives of activity'
+ogImage: 
+toc: false
+categories:
+  - Community
+---
+
+Author: Bill Mulligan - Isovalent
+
+Today the eBPF community is excited to announce the launch of the next version of [eBPF](https://ebpf.io/), excelBPF. Excel is the most widely used programming language in the world and underpins the functioning of many of the most critical institutions in the world. Bringing power of BPF to spreadsheets will enable millions of people to leverage this production proven technology, making bytecode only one cell away.
```
[the remark this answers is not in the document]
Comment (xmulligan) [reply content/blog-posts/2023-04-01-excelbpf/index.md:14]: Love it!

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+With money flowing this quickly through **excelBPF™**, you could easily guess that the funding for companies is just behind it and you wouldn’t be wrong. “After seeing how fast Isovalent changed the cloud native landscape with eBPF 1.0, we have an open checkbook for companies looking to build around **excelBPF™**. I'm literally just waiting for those **excelBPF™** financial models to hit my inbox.” said Martanique Casandro, Partner at Redwood Tree Venturez (r17z).
+
+The [eBPF Landscape](https://ebpf.io/applications/) is rapidly evolving and we are excited to see the first **excelBPF™** powered applications be added to the landscape. Be sure to check back regularly to see how the space is evolving. If you want to catch up on eBPF 1.0 before diving into excelBPF, check out [what is eBPF](https://ebpf.io/what-is-ebpf/), the [getting started page](https://ebpf.io/get-started/), our [interactive eBPF labs](https://play.instruqt.com/embed/isovalent/tracks/ebpf-getting-started?token=em_9nxLzhlV41gb3rKM&show_challenges=true), and subscribe to [eCHO News](https://ebpf.io/newsletter/) to keep up on all the updates from the eBPF community.
```
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:28]:
```suggestion
The [eBPF Landscape](https://ebpf.io/applications/) is rapidly evolving and we are excited to see the first **excelBPF™** powered applications be added to the landscape. Be sure to check back regularly to see how the space is evolving. If you want to catch up on eBPF 1.0 before diving into excelBPF, check out [what is eBPF](https://ebpf.io/what-is-ebpf/), the [getting started page](https://ebpf.io/get-started/), our [interactive eBPF labs](https://play.instruqt.com/embed/isovalent/tracks/ebpf-getting-started?token=em_9nxLzhlV41gb3rKM&show_challenges=true), and subscribe to [eCHO News](https://ebpf.io/newsletter/) to keep up on all the updates from the eBPF community.

![demo of excelBPF](excelBPF.mov)

_See excelBPF in action in this short demo from [Liz Rice](https://twitter.com/lizrice)_
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+![demo of excelBPF](excelBPF.mov)
```
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:30]:
```suggestion
<video width="320" height="240" controls>
  <source src="excelBPF.mov" type="video/mp4">
</video>
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 12]
```diff
@@ -27,6 +27,8 @@ With money flowing this quickly through **excelBPF™**, you could easily guess
 
 The [eBPF Landscape](https://ebpf.io/applications/) is rapidly evolving and we are excited to see the first **excelBPF™** powered applications be added to the landscape. Be sure to check back regularly to see how the space is evolving. If you want to catch up on eBPF 1.0 before diving into excelBPF, check out [what is eBPF](https://ebpf.io/what-is-ebpf/), the [getting started page](https://ebpf.io/get-started/), our [interactive eBPF labs](https://play.instruqt.com/embed/isovalent/tracks/ebpf-getting-started?token=em_9nxLzhlV41gb3rKM&show_challenges=true), and subscribe to [eCHO News](https://ebpf.io/newsletter/) to keep up on all the updates from the eBPF community.
 
-![demo of excelBPF](excelBPF.mov)
+<video width="320" height="240" controls>
+  <source src="excelBPF.mov" type="video/mp4">
+</video>
```
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:32]:
```suggestion
<video src="excelBPF.mov" width="1036" height="720" controls muted></video>
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+The [eBPF Landscape](https://ebpf.io/applications/) is rapidly evolving and we are excited to see the first **excelBPF™** powered applications be added to the landscape. Be sure to check back regularly to see how the space is evolving. If you want to catch up on eBPF 1.0 before diving into excelBPF, check out [what is eBPF](https://ebpf.io/what-is-ebpf/), the [getting started page](https://ebpf.io/get-started/), our [interactive eBPF labs](https://play.instruqt.com/embed/isovalent/tracks/ebpf-getting-started?token=em_9nxLzhlV41gb3rKM&show_challenges=true), and subscribe to [eCHO News](https://ebpf.io/newsletter/) to keep up on all the updates from the eBPF community.
+
+<video src="excelBPF.mp4" width="1036" height="720" controls muted></video>
```
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:30]:
```suggestion
<video src="/videos/excelBPF.mp4" type="video/mp4" width="1036" height="720" controls muted></video>. 
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+
+eBPF has been deployed at scale for over half a decade now, delivering you every cat picture on Facebook, processing and securing your Google queries, and enabling you to binge shows on Netflix without waiting for it to load. These initial adopters showed us the impact eBPF can have on performance and innovation in areas like networking, observability, security, tracing, and profiling. **excelBPF™** will now bring these benefits to many new use cases like finance, inventory management, consulting, and planning your next vacation.
+
+“When we got the first version of eBPF merged into the Linux kernel in 2014, we had ideas about how transformative the technology could be for the Linux kernel, but they were still to be proven out. Seeing the massive step change in innovation and flexibility eBPF has enabled for the kernel community, I’m excited to see what **excelBPF™** holds in store for the world. I think we are really just scratching the surface of what is possible. It feels like Day 1 all over again.” said Daniel Borkmann, co-creator of eBPF.
+
```
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:19]:
```suggestion

<video src="/videos/excelBPF.mp4" type="video/mp4" width="1036" height="720" controls muted></video>. 

```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+The [eBPF Landscape](https://ebpf.io/applications/) is rapidly evolving and we are excited to see the first **excelBPF™** powered applications be added to the landscape. Be sure to check back regularly to see how the space is evolving. If you want to catch up on eBPF 1.0 before diving into excelBPF, check out [what is eBPF](https://ebpf.io/what-is-ebpf/), the [getting started page](https://ebpf.io/get-started/), our [interactive eBPF labs](https://play.instruqt.com/embed/isovalent/tracks/ebpf-getting-started?token=em_9nxLzhlV41gb3rKM&show_challenges=true), and subscribe to [eCHO News](https://ebpf.io/newsletter/) to keep up on all the updates from the eBPF community.
+
+<video src="/videos/excelBPF.mp4" type="video/mp4" width="1036" height="720" controls muted></video>. 
+
+_See excelBPF in action in this short demo from [Liz Rice](https://twitter.com/lizrice)_
```
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:32]:
```suggestion
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+
+eBPF has been deployed at scale for over half a decade now, delivering you every cat picture on Facebook, processing and securing your Google queries, and enabling you to binge shows on Netflix without waiting for it to load. These initial adopters showed us the impact eBPF can have on performance and innovation in areas like networking, observability, security, tracing, and profiling. **excelBPF™** will now bring these benefits to many new use cases like finance, inventory management, consulting, and planning your next vacation.
+
+“When we got the first version of eBPF merged into the Linux kernel in 2014, we had ideas about how transformative the technology could be for the Linux kernel, but they were still to be proven out. Seeing the massive step change in innovation and flexibility eBPF has enabled for the kernel community, I’m excited to see what **excelBPF™** holds in store for the world. I think we are really just scratching the surface of what is possible. It feels like Day 1 all over again.” said Daniel Borkmann, co-creator of eBPF.
+
+<video src="/videos/excelBPF.mp4" type="video/mp4" width="1036" height="720" controls muted></video>. 
+
```
Comment (xmulligan) [inline content/blog-posts/2023-04-01-excelbpf/index.md:21]:
```suggestion
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 17]
```diff
@@ -0,0 +1,31 @@
+---
+path: '/blog/launching-excel-bpf'
+date: '2023-04-01T10:00:00.000Z'
+title: 'Annoucing eBPF 2.0: excelBPF'
```
Comment (qmonnet) [inline content/blog-posts/2023-04-01-excelbpf/index.md:4]:
```suggestion
title: 'Annoucing eBPF 2.0: excelBPF™'
```

File: content/blog-posts/2023-04-01-excelbpf/index.md [hunk 18]
```diff
@@ -0,0 +1,31 @@
...
+
+eBPF has been deployed at scale for over half a decade now, delivering you every cat picture on Facebook, processing and securing your Google queries, and enabling you to binge shows on Netflix without waiting for it to load. These initial adopters showed us the impact eBPF can have on performance and innovation in areas like networking, observability, security, tracing, and profiling. **excelBPF™** will now bring these benefits to many new use cases like finance, inventory management, consulting, and planning your next vacation.
+
+“When we got the first version of eBPF merged into the Linux kernel in 2014, we had ideas about how transformative the technology could be for the Linux kernel, but they were still to be proven out. Seeing the massive step change in innovation and flexibility eBPF has enabled for the kernel community, I’m excited to see what **excelBPF™** holds in store for the world. I think we are really just scratching the surface of what is possible. It feels like Day 1 all over again.” said Daniel Borkmann, co-creator of eBPF.
+
+<video src="/videos/excelBPF.mp4" type="video/mp4" width="1036" height="720" controls muted></video>.
```
Comment (qmonnet) [inline content/blog-posts/2023-04-01-excelbpf/index.md:20]:
```suggestion
<video src="/videos/excelBPF.mp4" type="video/mp4" width="1036" height="720" controls muted></video>
```

**Pull request merged**